README.md

```diff
@@ -25,37 +25,36 @@ P.S: When you import the Inspetor library you will see that you are actually ins
 You can find more in-depth documentation about our frontend libraries and integrations in general [here](https://inspetor.github.io/docs-frontend).
 
 ### Library setup
-In order to properly relay information to Inspetor's processing pipeline, you'll need to provide customer-specific authentication credentials:
-- App ID (provided by Inspetor)
-- Tracker name (provided by Inspetor)
+In order to properly relay information to Inspetor's processing pipeline, you'll need to provide your customer-specific authentication credential:
+- authToken (provided by Inspetor)
+
+**P.S:** Remember to use the *sandbox* `authToken` when you are not in production
 
 With these, you can instantiate the Inspetor tracking instance. Our integration library instantiates a singleton instance to prevent multiple trackers from being instantiated, which could otherwise result in duplicate or inconsistent data being relayed to Inspetor. Apart from that, the singleton will let you configure the library only once.
 
 The singleton instance is instantiated as follows:
 
 ```
 do {
-  try Inspetor.sharedInstance().setup(appId: "appId", trackerName: "trackerName", devEnv: false)
+  try Inspetor.sharedInstance().setup(authToken: "authToken")
 } catch TrackerException.requiredConfig(let code, let message) {
   print("code: \(code) - message: \(message)")
 } catch {
   print("Error")
 }
 ```
 
-Be advised, that this function can throw an exception if you pass an invalid appId or/and trackerName (empty strings or not in the format required).
+e advised, that this function can throw an exception if you pass invalids (empty strings or not in the format required) authToken.
```

Comment:
e advised --> Be advised

Also I guess it should be `pass an invalid authToken (empty strings or not in the format required).`